streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -184,10 +187,7 @@ public void shouldThrowLockExceptionIfFailedToLockStateDirectory() throws IOExce
 
     @Test
     public void shouldTransitToRunningAfterInitialization() {
-        EasyMock.expect(stateManager.changelogOffsets()).andStubReturn(Collections.emptyMap());
```

Comment:
Quite a lot of these setup statements have been removed as the default Mockito behaviour is to return empty collections and the strict stubbing detects them and asks for them to be removed.